### PR TITLE
Chloe

### LINKED ISSUE #7: Creating a dataset guidelines outline
We should work out an outline for these guidelines.
We could start by listing the main steps of a dataset creation

### PR DESCRIPTION
**Pull request by chloemackallah**
Added Introduction (set as root) and 'About Us' section which includes goals, WG governance document, and how-to-contribute page.

Resolves #7 (?)

### CODE REVIEW

**Comment by paolap** [comment]
@chloemackallah I like it,

a few comments:

I think goals should maybe appear after the Governance as it's a more detailed account of what we're trying to do. In fact it might not hurt to make that become some sort of timeline so we're not only telling people what we plan to do but also what we are more actively working on, what's complete.
As we are talking about our goals also in the governance, either this provide more detail or it becomes redundant. If we changing it then the title should also reflect the change.

In the how to contribute, it's really a minor thing but I would put fork/clone or is it clear that you can clone and work on a copy of the same repo locally?

File: Governance/about-us/how-to-contribute.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**No edits should be made to the main branch of ACDguide/Governance (the base repository), unless correcting minor typos.**
+
+#### Reviewing/commenting on PRs
+- Assign someone to review your PR. This can be done by clicking the gear in the upper right of the PR next to "Assignees". A drop-down menu of all maintainers should appear and you can choose the appropriate person or people.
+- Who should you ask to review your PR? It depends on the page you are updating.
+  - `Creating climate data`: Chloe (@chloemackallah)
+  - `Publishing climate data`: Paola (@paolap)
+  - `Updateing published data`: Claire (@hot007)
```
Comment (hot007) [inline Governance/about-us/how-to-contribute.md:30]: *Updating

File: Governance/about-us/how-to-contribute.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - `Retiring published data`: Claire? (@hot007)
+  - `Computations with large datasets`: Scott (@ScottWales)
```
Comment (hot007) [inline Governance/about-us/how-to-contribute.md:32]: Should this be a short blurb and a link to the relevant chapter in BigData?

File: Governance/introduction.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+This book, '**Climate Dataset Guidelines**', provides guidelines for creating, managing, sharing and publishing climate data. With a focus on Australian climate research it has been designed to enable a common approach across the community. Despite the focus on climate data, many of the principles and tools described within are applicable to many Earth science data applications.
+
+These guidelines are applicable to all levels of data sharing, from peer-to-peer sharing with colleagues to full publication in a public repository. If you expect that even a single other person will use your data for any purpose, including simple checking and comparing, your data should meet FAIR principles and meet a minimum set of formatting and metadata recommendations.
```
Comment (hot007) [inline Governance/introduction.md:5]: Could tone this down a little - all of FAIR is a bit much for a single collaborator! :) maybe "If you expect that even one other person will use your data for any purpose <...> your data should be structured appropriately considering aspects of FAIR principles and <...>"

File: Governance/introduction.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+These guidelines are applicable to all levels of data sharing, from peer-to-peer sharing with colleagues to full publication in a public repository. If you expect that even a single other person will use your data for any purpose, including simple checking and comparing, your data should meet FAIR principles and meet a minimum set of formatting and metadata recommendations.
+
+#
+This book covers the follow topics:
+
+### **Creating climate data**
+Creating datasets that are fit for publication can be a daunting process, with many technical aspects to consider. This chapter will outline the ways to ensure that your data meets the broad [FAIR principles](https://ardc.edu.au/resources/working-with-data/fair-data/) of data sharing, along with specific requrirements and recommendations for climate-related data that will aid in the use and reproducibility of your data by the Australian, and international, climate community.
+
+### **Publishing climate data**
```
Comment (hot007) [inline Governance/introduction.md:13]: These section headers could be URLs/anchor links pointing to the relevant pages?

**Review by hot007 — APPROVED** [review]
Happy to approve pending minor corrections (at least the updateing typo!).
LGTM :)

**Pull request merged**